src/OrbitGl/CaptureViewElement.h

```diff
@@ -24,7 +24,7 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
                               orbit_gl::Viewport* viewport, TimeGraphLayout* layout);
   virtual void Draw(Batcher& /*batcher*/, TextRenderer& /*text_renderer*/,
                     uint64_t /*current_mouse_time_ns*/, PickingMode /*picking_mode*/,
-                    float /*z_offset*/ = 0) {}
+                    uint32_t /*indentation_level*/ = 0, float /*z_offset*/ = 0) {}
```

Comment:
Not sure whether we should also add the `indentation_level` to the `CaptureViewElement`.

The meaning of `indentation_level` is obvious for the `Track::Draw` method but not very clear for the `CaptureViewElement::Draw` method.

Reply:
We also have to included because Draw() is a method shared with all Capture View Elements and needed in Track ones. It's the same with current_mouse_time_ns which is only used in GraphTrack. We made the DrawContext structure (Henning's idea) for hiding some parameters but if you have a better idea, feel free to tell me.

Reply:
Thanks for the explaination!